memory_profiler.py

```diff
@@ -987,33 +1083,38 @@ def flush(self):
     parser = OptionParser(usage=_CMD_USAGE, version=__version__)
     parser.disable_interspersed_args()
     parser.add_option(
-        "--pdb-mmem", dest="max_mem", metavar="MAXMEM",
-        type="float", action="store",
-        help="step into the debugger when memory exceeds MAXMEM")
+        '--pdb-mmem', dest='max_mem', metavar='MAXMEM',
+        type='float', action='store',
+        help='step into the debugger when memory exceeds MAXMEM')
     parser.add_option(
-        '--precision', dest="precision", type="int",
-        action="store", default=3,
-        help="precision of memory output in number of significant digits")
-    parser.add_option("-o", dest="out_filename", type="str",
-                      action="store", default=None,
-                      help="path to a file where results will be written")
-    parser.add_option("--timestamp", dest="timestamp", default=False,
-                      action="store_true",
-                      help="""print timestamp instead of memory measurement for
-                      decorated functions""")
+        '--precision', dest='precision', type='int',
+        action='store', default=3,
+        help='precision of memory output in number of significant digits')
+    parser.add_option('-o', dest='out_filename', type='str',
+                      action='store', default=None,
+                      help='path to a file where results will be written')
+    parser.add_option('--timestamp', dest='timestamp', default=False,
+                      action='store_true',
+                      help='''print timestamp instead of memory measurement for
```

Comment:
I think stylistic changes like replacing double- with single- quotes are better suited for a separate PR. Also, keep in mind that PEP-8 [advises against](http://pep8.org/#string-quotes) using triple-single-quotes:

> For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257.